core/src/main/java/org/nervos/ckb/service/RpcService.java

```diff
@@ -0,0 +1,129 @@
+package org.nervos.ckb.service;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.google.gson.reflect.TypeToken;
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicLong;
+import okhttp3.*;
+import okhttp3.logging.HttpLoggingInterceptor;
+import org.jetbrains.annotations.NotNull;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
+public class RpcService {
```

Comment:
Are `RpcService` and `RpcCallback` supposed to be used internally only? If so is there a way in Java to not mark them as `public`?

Please ignore this is users would call and use `RpcService` directly.

Reply:
I'm not sure, because SDK does not implement async request in `Api.java` and just provides a `postAsync` method in `RpcService.java`. 

So I set `RpcService` and `RpcCallback` public to support user can do async request if he has related requirement.  Of course, I also can set them to private. 

What do you think about this? @ashchan

Reply:
Let's mark them private for now then. We could expose more friendly async interfaces in the future if needed. For example:

Reply:
OK.  I will update later.